content/docs/studio/user-guide/model-registry/download-models.md

```diff
@@ -1,28 +1,75 @@
-versions using [dvc get](https://dvc.org/doc/command-reference/get).
+versions using `dvc get`.
+
+### Looking up right command in Studio
+
+Model details page have a command ready to copy and use under "Get the model
+file": ![](https://hackmd.io/_uploads/SkE5qUGr2.png)
+
+### Constructing the command manually
+
+You can also construct this command manually using this template:
 
 ```cli
 $ dvc get ${GIT_REPO} ${MODEL_PATH} --rev ${GIT_REV}
 ```
 
-For example, one of the models in the
-[bank customer churn demo project](https://github.com/iterative/demo-bank-customer-churn)
-is `lightgbm-model`. You can find its registered versions in the
-[list of Git tags](https://github.com/iterative/demo-bank-customer-churn/tags).
+Let's download the model in the
+[example-get-started-experiments](https://github.com/iterative/example-get-started-experiments)
+is `seg-model`.
 
-Suppose you want to dowload the model file for the `lightgbm-model@v2.0.1`
-version. In the model details page, you can see that the model file path is
-`.mlem/model/lightgbm-model`.
+To find out the model file path (e.g. `models/model.pkl`) you can see the
+Model's details page or check `dvc.yaml` the model annotation was written to.
 
-To download the file, you can use the following dvc command:
+To find out the Git revision, you can check the
+[list of Git tags](https://github.com/iterative/example-get-started-experiments/tags)
+or use [GTO](https://mlem.ai/doc/gto/command-reference/show/):
 
 ```cli
-$ dvc get https://github.com/iterative/demo-bank-customer-churn \
-    .mlem/model/lightgbm-model \
-    --rev lightgbm-model@v2.0.1
+$ gto show seg-model@latest --ref
+seg-model@v0.0.1#1
 ```
 
-Note that `.mlem/model/lightgbm-model` is the path where the model is registered
-using GTO, and not the physical path of the model file (in the remote storage).
+Putting everything together, to download the model file, you can use the
+following dvc command:
+
+```cli
+$ dvc get https://github.com/iterative/example-get-started-experiments \
+    models/model.pkl \
+    --rev seg-model@v0.0.1#1
+```
+
+Note that `models/model.pkl` is the path that was specified in `dvc.yaml`, and
+not the physical path of the model file (in the remote storage).
+
+DVC will figure out the right file to download from the remote storage for you.
+Note that for running this command, you will need to have remote storage
+credentials set up in your environment.
+
+## Deploying and publishing models in CI/CD
+
+A popular deployment option is to use CI/CD (triggered when Git tags are pushed)
+to publish new model version for consumers to use, or deploy it upon stage
+assignment.
+
+Since GTO register versions and assign stages by creating Git tags, you can set
+up a CI/CD to be triggered once they're pushed to the repo. To see an actual
+example, check out
+[the workflow](https://github.com/iterative/example-gto/blob/main/.github/workflows/gto-act-on-tags.yml)
```

Comment:
Example looks like a good start! One minor question: should we restrict the trigger to only tags that are in gto format so user's own tags don't also trigger it?

Reply:
Right! Created an issue to address it later. https://github.com/iterative/gto/issues/354